modules/statics/src/map.ts

```diff
@@ -44,7 +51,7 @@ export class CoinMap {
   }
 
   public has(key: string): boolean {
-    return this._map.has(key) || this._coinByAliases.has(key);
+    return this._map.has(key) || this._coinByIds.has(key) || this._coinByAliases.has(key);
```

Comment:
Can you kindly add a test case for this?

Reply:
done